README.md

```diff
@@ -71,8 +91,22 @@ The validation report can be exported in several ways with the `-r` / `--report`
 
 The reports written into a file are named after the input file, followed by a timestamp. The default output directory is the same as the input file's if provided using `-i`, or the current directory if using the standard input; it can be changed with the `-o` / `--outdir` option.
 
-### Examples
+### Debugulator
+
+There are some simple errors that can be automatically fixed. The most common error is the presence of duplicate variants. The needed parameters are the original VCF and the report generated by a previous run of the vcf_validator with the option `-r database`.
+ 
+The fixed VCF will be written into the standard output, which you can redirect to a file, or use the `-o` / `--output` option and specify the desired file name.
+
+The logs about what the debugulator is doing will be written into the error output. The logs may be redirected to a log file `2>debugulator_log.txt` or completely discarded ` 2>/dev/null`
+
+## Examples
 
 Simple example: `vcf_validator -i /path/to/file.vcf -v v4.1`  
 Full example: `vcf_validator -i /path/to/file.vcf -v 4.1 -l stop -r database,stdout -o /path/to/output/folder/`
```

Comment:
@garysaunders detected an error in this example: it should state `-v v4.1` instead of `-v 4.1`.